autoPyTorch/api/base_task.py

```diff
@@ -164,25 +168,25 @@ class BaseTask(ABC):
     """
 
     def __init__(
-        self,
-        seed: int = 1,
-        n_jobs: int = 1,
-        n_threads: int = 1,
-        logging_config: Optional[Dict] = None,
-        ensemble_size: int = 50,
-        ensemble_nbest: int = 50,
-        max_models_on_disc: int = 50,
-        temporary_directory: Optional[str] = None,
-        output_directory: Optional[str] = None,
-        delete_tmp_folder_after_terminate: bool = True,
-        delete_output_folder_after_terminate: bool = True,
-        include_components: Optional[Dict[str, Any]] = None,
-        exclude_components: Optional[Dict[str, Any]] = None,
-        backend: Optional[Backend] = None,
-        resampling_strategy: ResamplingStrategies = HoldoutValTypes.holdout_validation,
-        resampling_strategy_args: Optional[Dict[str, Any]] = None,
-        search_space_updates: Optional[HyperparameterSearchSpaceUpdates] = None,
-        task_type: Optional[str] = None
+            self,
```

Comment:
could you please revert the indentation here? I'll link this comment on other places it needs to be fixed.